src/pages/AdminCircleApiPage/AdminCircleApiPage.tsx

```diff
@@ -0,0 +1,114 @@
+import { useState } from 'react';
+
+import { useMutation, useQueryClient } from 'react-query';
+
+import { useSelectedCircleId } from '../../recoilState';
+import { ActionDialog, LoadingModal } from 'components';
+import { Box, Button, Modal, Panel, Text } from 'ui';
+import { OrgLayout } from 'ui/layouts';
+
+import { ApiKeyDisplay } from './ApiKeyDisplay';
+import { ApiKeyForm } from './ApiKeyForm';
+import { ApiKeyRow } from './ApiKeyRow';
+import { deleteCircleApiKey } from './mutations';
+import { useCircleApiKeys } from './useCircleApiKeys';
+
+const AdminCircleApiPage = () => {
+  const [modal, setModal] = useState<'' | 'create' | 'display'>('');
+  const [keyToDelete, setKeyToDelete] = useState('');
+  const [displayedKey, setDisplayedKey] = useState('');
+
+  const queryClient = useQueryClient();
+
+  const circleId = useSelectedCircleId();
```

Comment:
Tried it, seems to be working fine. A bunch of places in the code are still using `useSelectedCircle`, which uses `useSelectedCircleId`. I just used it directly here since I dont need any of the other data that `useSelectedCircle` was assembling